sdcm/sct_config.py

```diff
@@ -1516,6 +1516,13 @@ class SCTConfiguration(dict):
 
         dict(name="validate_large_collections", env="SCT_VALIDATE_LARGE_COLLECTIONS", type=boolean,
              help="Enable validation for large cells in system table and logs"),
+
+        dict(name="kafka_backend", env="SCT_KAFKA_BACKEND", type=str,
+             help="Enable validation for large cells in system table and logs",
+             choices=("none", "localstack", "vm", "msk")),
+
+        dict(name="kafka_connectors", env="SCT_KAFKA_CONNECTORS", type=str_or_list_or_eval,
+             help="configuration for setup up kafka connectors"),
```

Comment:
Would be good to know where to get the config structure

Reply:
I think we'll point to one or two examples
also I introduce a pydantic dataclass hat would be able to verify this part of configuration

Reply:
pydantic dataclass was introduced, it should be a bit clearer now